ports/pic16bit/Makefile

```diff
@@ -19,7 +19,7 @@ INC += -I$(XC16)/include
 INC += -I$(XC16)/support/$(PARTFAMILY)/h
 
 CFLAGS_PIC16BIT = -mcpu=$(PART) -mlarge-code
-CFLAGS = $(INC) -Wall -Werror -std=gnu99 -nostdlib $(CFLAGS_PIC16BIT) $(COPT)
+CFLAGS = $(INC) -Wall -Wfloat-conversion -Werror -std=gnu99 -nostdlib $(CFLAGS_PIC16BIT) $(COPT)
```

Comment:
This port doesn't enable floats, and not sure the PIC compiler would support this flag (?).